src/config.test.ts

```diff
@@ -1,21 +1,31 @@
-import config, { DEFAULT_BLOCK_GAS_LIMIT } from './config';
+import config, { DEFAULT_BLOCK_GAS_LIMIT, Config } from './config';
 
-const defaultConfig = {
+const defaultConfig: Config = {
   accounts: {
     amount: 10,
     ether: 100,
   },
+
   contracts: {
     type: 'truffle',
     defaultGas: DEFAULT_BLOCK_GAS_LIMIT * 0.75,
+    defaultGasPrice: 20e9, // 20 gigawei
+    artifactsDir: 'build/contracts',
   },
 
   blockGasLimit: DEFAULT_BLOCK_GAS_LIMIT,
-  gasPrice: 20e9,
+  gasPrice: 20e9, // 20 gigawei
+
+  setupProvider: async baseProvider => baseProvider,
+
+  coverage: false,
+
+  node: { allowUnlimitedContractSize: false },
 };
 
 describe('config', (): void => {
   it('provides default value', (): void => {
-    expect(config).toMatchObject(defaultConfig);
+    // because setupProvider wouldn't match with toEqual
+    expect(JSON.stringify(config)).toBe(JSON.stringify(defaultConfig));
```

Comment:
Note that `JSON.stringify` doesn't include the `setupProvider` field in its output because serializing functions is not supported. Was that the desired outcome?

Reply:
Yes, because if you include it a strict check would fail. `toMatchObject` on the other hand allows to pass check which are missing some properties.